### PR TITLE
Chat cosmetic change

### PR DESCRIPTION
**Pull request by VictorTennekes**
Made some visual changes to the chat like message formatting, adding a send icon and some color updates :)

Screen Shot 2021-09-28 at 11 50 15 AM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (saarikoski-jules) [inline package.json:1]: I don't think we need this file?

**Review by saarikoski-jules — CHANGES_REQUESTED** [review]
Looks awesome! There are two small changes I'd like to request:

- I don't think we need a package.json in the root of the project. If it's not needed, would you remove it?
- when a chat is a DM it doesn't have a name, so the chat messages will be hidden under the "home" symbol. Would you either add a name for DM's or make sure the name component will render space even when nameless.

Otherwise, great job!

**Comment by VictorTennekes** [comment]
Should be fixed now :)

**Review by saarikoski-jules — APPROVED** [review]
wow you're defs the most hardworking one of us all

**Pull request merged**